backend/src/Controllers/AttachmentsController.php

```diff
@@ -163,13 +185,37 @@ public function count( WP_REST_Request $request ) {
 	 *
 	 * @return mixed|WP_REST_Response
 	 */
-	public function index( WP_REST_Request $request ) {
+	public function index(WP_REST_Request $request)
+	{
 
 		$args = $request->get_params();
 
-		return $this->attachments->paginate( $args )
-								 ->apply_transform( $this->transformer )
-								 ->to_rest_response();
+		switch ($args['post_mime_type']) {
+			case 'spreadsheets':
+				$args['post_mime_type'] = 'pplication/vnd.apple.numbers,application/vnd.oasis.opendocument.spreadsheet,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,application/vnd.ms-excel.sheet.macroEnabled.12,application/vnd.ms-excel.sheet.binary.macroEnabled.12';
```

Comment:
might be a missing "a" here: `= 'pplication/...`

Reply:
Added missed 'a'